sql/handler.h

```diff
@@ -1738,6 +1671,12 @@ typedef bool (*rotate_encryption_master_key_t)(void);
 */
 using fix_tablespaces_empty_uuid_t = bool (*)(void);
 
+using compression_dict_data_vec_t =
+    std::vector<std::pair<std::string, std::string>>;
```

Comment:
I think there is little value with allocator in this case because server is in upgrade mode. connections are not accepted at this stage. We cannot see the PFS data either.